utils/scripts/terraform/openstack/README.md

```diff
@@ -0,0 +1,21 @@
+# Provision vm in openstack
+Script to provision vm in openstack using terraform
+
+## Prerequisites
+Python 3.x
+Install terraform in cli
+pip3 install python-terraform
+
+## How to Run:
+1. Change directory to the terraform/openstack directory.
+
+```
+# cd ods-ci/utils/scripts/terraform/openstack
+```
+
+2. Now run the script.
+
+## Usage
+
+```
+# python3 provision.py -h
```

Comment:
Can you add an example with all the required parameters?

Reply:
Done.